config/load.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"io/ioutil"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/kelseyhightower/envconfig"
+)
+
+type ConfigPath struct {
+	FromYAML   bool   `long:"fromYaml" env:"FROM_YAML" description:"load configuration from yaml file otherwise from ENV"`
```

Comment:
presence of the yaml config file should define if yaml is used or not. it shouldnt' be driven by another variable